genon/preprocessor/tests/unit/test_enrichment_yaml_unit.py

```
@@ -0,0 +1,284 @@
+        assert ec_on.image_description_cfg.get("enabled") is True
+
+        ec_off = EnrichmentConfig.from_raw(
+            [{"image_description": {"enable": False}}],
+            Path("."),
+        )
+        assert ec_off.image_description_cfg == {"enabled": False}
+
+    def test_single_custom_fields_injects_resource_path(self):
+        ec = EnrichmentConfig.from_raw(
+            [{"custom_fields": {"enable": True, "output_fields": ["authors"], "system_prompt": "s"}}],
+            Path("/tmp/cfgdir"),
```

Ruff (0.15.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/cfgdir"

(S108)